<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBoxConnecServ">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>611</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion au serveur</string>
   </property>
   <widget class="QLabel" name="labelAdIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP du serveur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNroPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num√©ro de port du serveur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEtatConnec">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIPServ">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNroPort">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditEtatConnec">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnec">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxInfoClient">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>611</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations Client</string>
   </property>
   <widget class="QLineEdit" name="lineEditNomOrdi">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>411</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomUser">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>411</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAchiProces">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>411</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOSOrdi">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>411</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOrdinateur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'ordinateur distant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUtilisateur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'utilisateur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonArchitecture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Architecture proceseur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OS de l'ordinateur</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>ClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
